packages/@aws-cdk/aws-lambda/lib/docker.ts

```diff
@@ -0,0 +1,141 @@
+import { spawnSync } from 'child_process';
+
+/**
+ * A Docker volume
+ */
+export interface DockerVolume {
+  /**
+   * The path to the file or directory on the host machine
+   */
+  readonly hostPath: string;
+
+  /**
+   * The path where the file or directory is mounted in the container
+   */
+  readonly containerPath: string;
+}
+
+/**
+ * Docker run options
+ */
+export interface DockerRunOptions {
+  /**
+   * The command to run in the container.
+   */
+  readonly command: string[];
+
+  /**
+   * Docker volumes to mount.
+   *
+   * @default - no volumes are mounted
+   */
+  readonly volumes?: DockerVolume[];
+
+  /**
+   * The environment variables to pass to the container.
+   *
+   * @default - no environment variables.
+   */
+  readonly environment?: { [key: string]: string; };
+
+  /**
+   * Working directory inside the container.
+   *
+   * @default - image default
+   */
+  readonly workingDirectory?: string;
+}
+
+/**
+ * Docker build options
+ */
+export interface DockerBuildOptions {
+  /**
+   * Build args
+   *
+   * @default - no build args
+   */
+  readonly buildArgs?: { [key: string]: string };
+}
+
+/**
+ * A Docker image
+ */
+export class DockerImage {
+  /**
+   * Use an existing Docker image
+   *
+   * @param image the image name
+   */
+  public static fromImage(image: string) {
+    return new DockerImage(image);
+  }
+
+  /**
+   * Build a Docker image
```

Comment:
Fix the comments to reflect the relationship to bundling. Something like "Execute the bundling commands inside a docker image built from a local docker file"